app/viewModels/ViewArrival.scala

```diff
@@ -23,7 +23,11 @@ import play.api.libs.json.{JsObject, Json, OWrites}
 import java.time.format.DateTimeFormatter
 import java.time.{LocalDate, LocalTime}
 
-final case class ViewArrival(date: LocalDate, time: LocalTime, movementReferenceNumber: String, status: String, actions: Seq[ViewMovementAction])
+final case class ViewArrival(updatedDate: LocalDate, updatedTime: LocalTime, movementReferenceNumber: String, status: String, actions: Seq[ViewMovementAction])
```

Comment:
Is this view model passed to any other views? If it is then those pages would break silently since the views are not compiled

Reply:
Have tested branch manually with no issues.